app/scripts/lib/setupSentry.js

```diff
@@ -142,19 +142,19 @@ function getTracesSampleRate(sentryTarget) {
 }
 
 /**
- * Get CircleCI tags passed from the test environment, through manifest.json,
+ * Get CI tags passed from the test environment, through manifest.json,
  * and give them to the Sentry client.
  */
-function setCircleCiTags() {
-  const { circleci } = getManifestFlags();
+function setCITags() {
+  const { ci } = getManifestFlags();
 
-  if (circleci?.enabled) {
-    Sentry.setTag('circleci.enabled', circleci.enabled);
-    Sentry.setTag('circleci.branch', circleci.branch);
-    Sentry.setTag('circleci.buildNum', circleci.buildNum);
-    Sentry.setTag('circleci.job', circleci.job);
-    Sentry.setTag('circleci.nodeIndex', circleci.nodeIndex);
-    Sentry.setTag('circleci.prNumber', circleci.prNumber);
+  if (ci?.enabled) {
+    Sentry.setTag('ci.enabled', ci.enabled);
+    Sentry.setTag('ci.branch', ci.branch);
+    Sentry.setTag('ci.commitHash', ci.commitHash);
+    Sentry.setTag('ci.job', ci.job);
+    Sentry.setTag('ci.matrixIndex', ci.matrixIndex);
+    Sentry.setTag('ci.prNumber', ci.prNumber);
   }
 }
```

Comment:
I don't think this stuff is working. I wrote it originally, but it's a little tricky to test and debug.  I'll work on it.

Reply:
No, nobody is using this right now, not a blocker